src/server/components/user-name.jsx

```diff
@@ -21,7 +21,7 @@ const UserNameContainer = styled("a")`
   }
 `;
 
-const Avatar = ({ src }) => <img src={src} alt={src} />;
+const Avatar = ({ src }) => <img src={`${src}&s=30`} alt={src} />;
```

Comment:
It might make sense to define the `30` in `src/shared/theme` as something like `THEME.defaultUserIconSize`. @hamilton might have opinions/suggestions on this.